src/components/sections/challenges/_partials/HackathonPrize.tsx

```diff
@@ -1,12 +1,15 @@
 import { Distribution, Reward } from "@/types/course";
 import { shortenNumber } from "@/utilities";
+import { useTranslation } from "react-i18next";
 
 export default function HackathonPrize({ reward, description }: { reward: Reward; description: string }) {
+  const { t } = useTranslation()
   const { first, second, third } = reward?.distribution || ({} as Distribution);
+  const totalPrize = `${shortenNumber(reward?.amount)} ${reward?.fiatCurrency ? `${reward?.fiatCurrency} in` : ''} ${reward.token} ${t("communities.overview.reward.prize.pool")}`;
```

Comment:
Can you improve it this way?
```json
// /public/locale/common/json

"communities.overview.reward.prize.pool": "{{amount}} Prize pool",
```

```suggestion
  const totalPrize = `${shortenNumber(reward?.amount)} ${reward?.fiatCurrency ? `${reward?.fiatCurrency} in` : ''} ${reward.token}`;
  
  const prizePool = t("communities.overview.reward.prize.pool", { amount: totalPrize })
```